src/CloudEventFunctionWrapper.php

```diff
@@ -42,23 +42,37 @@ class CloudEventFunctionWrapper extends FunctionWrapper
 
     public function execute(ServerRequestInterface $request): ResponseInterface
     {
+        $body = (string) $request->getBody();
+        $jsonData = json_decode($body, true);
```

Comment:
CloudEvent body doesn't have to be JSON for binary CloudEvent.

Reply:
What else would it be? The current implementation of this requires JSON, I didn't change that in this PR...

Reply:
Added conditional validation for binary cloud events when the content-type ends in `json`